avaframe/in3Utils/geoTrans.py

```diff
@@ -547,6 +541,75 @@ def findSplitPoint(avaProfile, Points):
     return projPoint
 
 
+def findClosestPoint(xcoor, ycoor, pointsDict):
+    """ find closest point of pointDict along line defined by xcoor and ycoor - only xy plane!
```

Comment:
Maybe add info that pointsDict  has to contain 'x' and 'y'

Reply:
added